src/lib-components/lists/DetailList.vue

```diff
@@ -48,47 +47,60 @@ const loadAndRender = (checkForContent: boolean): void => {
         totalPages: success.data.totalPages,
       }
       items.value = success.data.items
-      if (checkForContent) hasContent.value = items.value.length != 0
     },
     () => {
       useAppFlasher.genericError()
     }
   )
 }
 
-watch([sortDir, dateRange], () => {
-  loadAndRender(false)
+const hasContent = computed((): boolean => {
+  return items.value.length ? true : false
 })
 
-watch(
-  () => props.refreshTrigger,
-  () => {
-    loadAndRender(false)
-  }
-)
-
-watch(
-  () => props.reloadTrigger,
-  () => {
-    // This lets parent components trigger a refresh of the current page depending on external actions.
-    pagination.value.page = 1
-    loadAndRender(true)
-  }
-)
+const filtersAreConfigured = computed(() => {
+  return props.config.dateSearch
+})
 
-loadAndRender(true)
+const showPaginator = computed(() => {
+  return !props.config.alwaysHideNav && hasContent
+})
+
+const reloadTable = (): void => {
+  pagination.value.page = 1
+  loadAndRender()
+}
+
+const publicMethods: DetailListAPI = {
+  refresh: loadAndRender,
+  reset: reloadTable,
+}
+
+watch([sortDir, dateRange], () => {
+  loadAndRender()
+})
+
+defineExpose(publicMethods)
+loadAndRender()
 </script>
 <template>
-  <div>
-    <DateFilter
-      :date-range="dateRange"
-      :sort-dir="sortDir"
-      :title="title"
-      @sort-dir-changed="sortDir = $event"
-      @date-range-changed="dateRange = $event"
-    />
+  <div :class="{ 'mt-4 space-y-2': filtersAreConfigured }">
+    <div v-if="config.dateSearch" class="flex">
+      <DateFilter
+        :date-range="dateRange"
+        :sort-dir="sortDir"
+        @sort-dir-changed="sortDir = $event"
+        @date-range-changed="dateRange = $event"
+      />
+    </div>
 
-    <div v-if="hasContent" class="shadow overflow-hidden sm:rounded-md border">
+    <div
+      v-if="hasContent"
+      class="overflow-hidden"
+      :class="{
+        'shadow sm:rounded-md border': filtersAreConfigured,
```

Comment:
We should make the shadow and border a configuration prop.  I'm trying to be less afraid of them, but it's a more stable and reliable option than letting callers pass class names into components.  We can do that in a pinch, but we shouldn't design components around that pattern.  We want to be able to make style changes without feeling like we have to go and examine every use case to see if there's conflicting `class` attribute being inherited.

I like the idea behind considering whether the existence of filters should drive the styling, but I'm less confident that will give us a meaningful runway.

`borderless?: bool` with a default of false seems like a sane place to start, I'd even apply it to the `li`, until we can narrow down how best to make this more configurable.